examples/src/main/java/com/mapbox/navigation/examples/activity/SimpleMapboxNavigationKt.kt

```diff
@@ -165,6 +196,27 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
+    private val fasterRouteSelectionTimer: CountDownTimer = object : CountDownTimer(startTimeInMillis, countdownInterval) {
+        override fun onTick(millisUntilFinished: Long) {
+            Timber.e("FASTER_ROUTE: millisUntilFinished $millisUntilFinished")
+            faster_route_accept_progress.progress = (maxProgress - millisUntilFinished / countdownInterval).toInt()
+        }
+
+        override fun onFinish() {
+            Timber.e("FASTER_ROUTE: finished")
+            this@SimpleMapboxNavigationKt.fasterRoute = null
+            bottomSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+        }
+    }
+
+    private val fasterRouteObserver = object : FasterRouteObserver {
```

Comment:
Do we need this as part of this PR? Currently faster route is disable by default downstream https://github.com/mapbox/mapbox-navigation-android/blob/fbbeee4685eaa8e156c84eac014471354fe7c058/libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/MapboxDirectionsSession.kt#L116-L119

Reply:
Yes. I would like to stick it as a part of this PR because that's the objective of this ticket. To be able to provide a callback for user to act on faster route. The part where we fix the above `isRouteFaster` is going to come up [here](https://github.com/mapbox/navigation-sdks/issues/221)